demo/startRedis.sh

```diff
@@ -0,0 +1,9 @@
+redis-server --daemonize yes && sleep 1
```

Comment:
It would be good to put some comments here explaining what is being done.

Reply:
Done